packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -56,6 +57,11 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     });
 
     if (!invitee) {
+      // if input is a username, throw error
+      if (!isEmail(usernameOrEmail)) {
+        throwUsernameDoesNotExistInOrgError(usernameOrEmail);
+      }
+
```

Comment:
checkInputEmailIsValid on L65 does this same thing

Reply:
@Udit-takkar, Thank you for finding out.
I just removed checkInputEmailIsValid as it is not required.
If the input is not a valid email, the code above will assume it as a username.